src/nrpe.c

```diff
@@ -980,13 +980,15 @@ int read_config_file(char *filename)
 			else
 				debug = FALSE;
 
-		} else if (!strcmp(varname, "nrpe_user"))
+		} else if (!strcmp(varname, "nrpe_user")) {
+			free(nrpe_user);
 			nrpe_user = strdup(varvalue);
 
-		else if (!strcmp(varname, "nrpe_group"))
```

Comment:
As a maintainer of other OSS project, personally I dislike such changes, which made without explicit needs. Merging such changes breaks other pull requests and [local] patches of project users, so forces all these people to do extra work just because you like other style of formatting. Also, these changes spray attention of reviewers, while in overall your proposals deserve attention. 

Thanks for your work, I found it useful for me.

Reply:
I'm confused as to the issue here.

This specific change is required to free any previously `strdup()` config value if it encounters another `nrpe_user` option in the config file or during configuration reload, otherwise we'll leak memory.

If you're referring to the brackets, those are also required as we're switching from a single statement to a compound statement block after the `if` clause.

Reply:
You completely correct, and I'm completely wrong.
Looking into code yesterday, I saw only brakets and nothing more. Sorry.

//I thought that was only cosmetic change, such as moving bracket to same line with 'if'.